Content.Server/GameObjects/EntitySystems/HandsSystem.cs

```diff
@@ -48,14 +48,19 @@ public override void Initialize()
                 .Register<HandsSystem>();
         }
 
+        private void HandleExamined(EntityUid uid, HandsComponent component, ExaminedEvent args)
+        {
+            foreach (var inhand in component.GetAllHeldItems())
+            {
+                args.Message.AddText($"\n{Loc.GetString("comp-hands-examine", ("user", component.Owner), ("item", inhand.Owner))}");
```

Comment:
i think i'd prefer having it on one line i.e. "They are holding an emergency crowbar and a popcorn box in their hands." but since mobs can have an arbitrary amount of hands that makes formatting the sentence harder so having it on multiple lines is ok right now